src/main/java/tornadofx/Menu.kt

```diff
@@ -46,6 +46,13 @@ fun ContextMenu.menuitem(name: String, keyCombination: KeyCombination? = null, g
     return menuItem
 }
 
+/**
+ * Add a separator to the contextmenu
+ */
+fun ContextMenu.separator(){
```

Comment:
I must have missed that. I'll take a closer look later but feel free to make the edit if you want.